packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -245,6 +259,21 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
     widget.onExpansionChanged?.call(_isExpanded);
   }
 
+  ListTileControlAffinity effectiveAffinity(ListTileControlAffinity affinity) {
+    if (affinity == ListTileControlAffinity.platform)
+      return ListTileControlAffinity.trailing;
+    return affinity;
+  }
+
+  Widget? _buildIcon(BuildContext context, ListTileControlAffinity affinity) {
```

Comment:
This method appears to exist to prevent two expansion arrows being created. A comment that explained what's going on would be helpful here.

Reply:
I changed the code so that instead of passing the affinity, there's `_buildLeadingIcon()` and `_buildTrailingIcon()` that are easier to understand, and it also leads to cleaner code at the call site.